### DISCUSSION TITLE
Intellij database results shared as snippets ?

### DISCUSSION THREAD

**Discussion by akulinski** [community/community#131006] [Copilot Conversations] [answered]
Hello I was wondering does copilot share IntelliJ database query results with GitHub ?

**Comment by Akash1134** [comment, marked as answer]
Hi @akulinski , GitHub Copilot does not share IntelliJ database query results with GitHub. Copilot processes personal data based on how Copilot is accessed and used: whether through features like suggestions for the command line interface (CLI), IDE code completions, or personalized chat on GitHub.com. The types of personal data processed may include:

- User Engagement Data: This includes pseudonymous identifiers captured on user interactions with Copilot, such as accepted or dismissed completions, error messages, system logs, and product usage metrics. 

- Prompts: These are inputs for chat or code, along with context, sent to Copilot's AI to generate suggestions. 

- Suggestions: These are the AI-generated code lines or chat responses provided to users based on their prompts. 

- Feedback Data: This comprises real-time user feedback, including reactions (e.g., thumbs up/down) and optional comments, along with feedback from support tickets.

Hope that helps!

**Reply by akulinski** [reply]
Thanks for confirmation
